package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "terra-enzyme-intl",
+  "version": "0.0.0",
+  "description": "Enzyme helpers for fortifying tests that depend on react-intl by decoupling the need for actual translations.",
+  "main": "lib/index.js",
+  "engines": {
+    "node": ">=8.9.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-enzyme-intl.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-enzyme-intl",
+    "jest",
+    "enzyme",
+    "react",
+    "react-intl",
+    "i18n",
+    "intl"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/cerner/terra-enzyme-intl#readme",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-enzyme-intl/issues"
+  },
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra"
+  },
+  "files": [
+    "lib"
+  ],
+  "dependencies": {
+    "browserslist-config-terra": "^1.1.0"
```

Comment:
This should be a dev dependency.

Reply:
https://github.com/cerner/terra-enzyme-intl/pull/2/commits/013c120bcea4351d2810bcaaa14d5acde14dc233